src/DynamoCore/Search/Legacy/BrowserRootElement.cs

```diff
@@ -23,6 +25,13 @@ public override string Name
             get { return _name; }
         }
 
+        /// <summary>
+        /// Specifies whether or not Root Category package / packages need update.
+        /// Makes sense for packages only.
+        /// TODO: implement as design clarifications are provided.
+        /// </summary>
+        public bool IsUpdateAvailable { get; set; }
```

Comment:
Is this `TODO` still applicable? If so please make sure there is a task written to follow up on this and insert the link here in comment.